src/azure-cli/azure/cli/command_modules/rdbms/_helptext_mysql.py

```diff
@@ -454,3 +454,11 @@
   - name: List log files for 'testsvr' less than 30Kb in size.
     text: az mysql flexible-server server-logs list -g testgroup -s testsvr --max-file-size 30
 """
+
+helps['mysql flexible-server upgrade'] = """
+type: command
+short-summary: Upgrade the major version of a flexible server.
+examples:
+  - name: Upgrade server 'testsvr' to MySQL major version 8.
```

Comment:
If this step is required, can we give instruction to customers somewhere? 
az mysql flexible-server parameter set -g testgroup -s testsvr -n sql_mode -v "ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO"

Reply:
Ok, I'll add a note about that in the help.